data/ka'het missions.txt

```diff
@@ -110,13 +111,16 @@ mission "First Contact: Ka'het: Quarg 1"
 			`	Apart from the utter silence, which is pretty common in most rings, you are convinced by the fact that every room, apartment and cabin has been carefully emptied, leaving nothing behind. Perhaps it would be wise to ask the Quarg what happened here...`
 				accept
 
-mission "First Contact: Ka'het: Quarg 2"
+# The Z ensures this happens after meeting the Quarg.
+mission "First Contact: Z Ka'het: Quarg 2"
 	landing
 	invisible
-	source "Lagrange"
+	source
+		attributes "quarg"
 	to offer
-		has "First Contact: Ka'het: Quarg 1: done"
+		has "First Contact: Ka'het: Quarg 1: active"
 	on offer
+		fail "First Contact: Ka'het: Quarg 1"
```

Comment:
Perhaps don't fail this in the `on offer`, but the `on decline`, so that if the player defers this mission, they still have the mission telling them to ask the Quarg about the ringworld.

Reply:
Yes.  I will retest with this change and make sure it still works.

It takes a while to get my test Sparrow to Ka'het space and then back to the Quarg.  The amusement value of using a Sparrow for this is worth the time.

Reply:
Fixed.